WISE-Community/WISE-Client
# chore(Classroom Monitor): Convert Nav Item

## Changes

Converted Classroom Monitor Nav Item to Angular.

## Test

Make sure the Grade By Step view works like before
- Make sure you can expand and collapse activities
- Make sure expanding an activity automatically scrolls to it
- Make sure locking and unlocking an activity works
- Make sure percentages show up on activities and steps
- Make sure rubric icons show up on steps
- Make sure score averages show up on steps

Closes #861
